src/main/java/Duke.java

```diff
@@ -1,10 +1,104 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        startDuke();
+
+        Scanner userInput = new Scanner(System.in);
+        String line;
+        ArrayList<Task> tasks = new ArrayList<Task>();
+
+        line = userInput.nextLine();
+        while (!"bye".equals(line)) {
+            Parse parsedInput = new Parse(line);
+            if (parsedInput.getIsKeyword()) {
+                switch (parsedInput.getParsedInput()[0]) {
+                case "todo":
+                    line = line.substring(5);
+                    Todo newTodo = new Todo(line);
+                    tasks.add(newTodo);
+                    System.out.println(wrapText("Added to new thing to do for ya! " + line));
+                    break;
+                case "deadline":
+                    String deadlineTask = line.substring(9, line.indexOf("/") - 1);
+                    String by = line.substring(line.indexOf("/") + 4);
+                    Deadline newDeadline = new Deadline(deadlineTask, by);
+                    tasks.add(newDeadline);
+                    System.out.println(wrapText("Added to thing for ya, and ya gotta do it soon! " + deadlineTask));
```

Comment:
Perhaps we can put the `println` as part of the `wrapText` method since the method encases one print?